### ISSUE TITLE
Reported test execution time is slower with --parallel flag

### ISSUE THREAD

**Issue by Iliev88** [cucumber/cucumber-js#1202]
I am wondering why time reported for tests execution with a parallel flag (e.g. --parallel 5) is significantly much than the time when tests are executed on a single process.

**with 5 slaves:**
```
35 scenarios (35 passed)
102 steps (102 passed)
0m48.478s
```

**without parallelism:**
```
test: 17628.303ms
35 scenarios (35 passed)
102 steps (102 passed)
0m17.550s
```

I've tried to measure the time with console.time() in BeforeAll/AfterAll hooks for each slave and the results are pretty interesting:

**with 5 slaves:**
```
test: 8718.911ms
test: 9902.674ms
test: 9668.365ms
test: 10364.837ms
test: 10052.901ms
```

**without parallelism:**
```
test: 17628.303ms
```

**The reported test execution time is a sum of the times for every separate node:**
```
test: 10769.271ms
test: 10991.144ms
test: 11037.363ms
test: 11875.058ms
test: 12177.004ms
35 scenarios (35 passed)
102 steps (102 passed)
0m56.636s
```

I am suspecting you are trying to explain it in the following part of the [README.md](https://github.com/cucumber/cucumber-js/blob/master/docs/cli.md#parallel-experimental)
`The reported runtime from the summary formatter is the total time from running the steps and thus be higher than the runtime for the command. The command runtime can be measured with other tools (time / Measure-Command)`

I'll be more than glad if I can receive a more detailed explanation and I just want to know that my tests are executed in a way allowing speeding up test execution once the project becomes bigger and bigger.

Cucumber: 5.1.0
Node: 12.2.0

Regards,
Iliya

**Comment by DevSide** [comment]
Yes it's a bug, the total execution time in parallel looks like the sum of each slave execution time.

**Comment by davidjgoss** [comment]
The summary formatter output has now been made clearer so it tells you both elapsed real time and aggregate time of running steps (excluding cucumber's internals):

![image](https://user-images.githubusercontent.com/3192745/100524284-b2df9600-31ae-11eb-95d2-22f0e3ebf259.png)

This will be available shortly in 7.0.0.

**Comment by davidjgoss** [comment]
7.0.0 is now released, see [the changelog](https://github.com/cucumber/cucumber-js/blob/master/CHANGELOG.md) for migration instructions. Thanks!
